apps/challenges/urls.py

```diff
@@ -39,6 +39,11 @@
         views.get_all_challenges,
         name="get_all_challenges",
     ),
+    url(
+        r"^challenge/participated/(?P<challenge_time>[A-Za-z]+)$",
```

Comment:
`r"^challenge/participated/(?P<challenge_time>[A-Za-z]+)$",` -> `r"^challenges/participated/(?P<challenge_time>[A-Za-z]+)/$",`

Reply:
@RishabhJain2018 I have made the suggested changes, please look at it.